contrib/ec2/deis.template.json

```diff
@@ -78,6 +78,13 @@
       "AllowedValues" : [ "PV", "HVM" ],
       "ConstraintDescription" : "must be either PV or HVM"
     },
+    "EC2EBSVolumeType" : {
+      "Description" : "EC2 EBS VolumeType",
+      "Type": "String",
+      "Default": "gp2",
+      "AllowedValues" : [ "gp2", "io1", "standard" ],
```

Comment:
Yep, you are absolutely correct. We could wrap that in an Fn::If. Will need to TDD that. Have some rspec ready (really need to blog about it soon), will give it a try.